Description: Correct escaping of characters
Author: Luca Bedogni <me@lucabedogni.it>

--- a/src/gtkcompletionline.cc
+++ b/src/gtkcompletionline.cc
@@ -226,12 +226,9 @@
   const char* i = str.c_str();
   while (*i) {
     char c = *i++;
-    switch (c) {
-     case ' ':
-      res += '\\';
-     default:
-      res += c;
-    }
+    if (c == ' ' || c == '(' || c == ')' || c =='\'')
+        res += '\\';
+    res += c;
   }
   return res;
 }
