Description: include missing headers
Author: Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>

--- a/src/ci_string.h
+++ b/src/ci_string.h
@@ -8,6 +8,7 @@
 
 #include <string>
 #include <ctype.h>
+#include <cstring>
 
 struct ci_char_traits : public std::char_traits<char>
 {
--- a/src/gtkcompletionline.cc
+++ b/src/gtkcompletionline.cc
@@ -30,6 +30,7 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <cstring>
 using namespace std;
 
 #include "gtkcompletionline.h"
