Description: Don't create an empty history file when History=0
Author: <bdefreese@bddebian3.bddebian.com>

--- a/src/history.cc
+++ b/src/history.cc
@@ -65,17 +65,19 @@
   if (!configuration.get_int("History", HIST_MAX_SIZE))
     HIST_MAX_SIZE = 20;
 
-  ofstream f(filename, ios::out);
+  if (HIST_MAX_SIZE) {
+    ofstream f(filename, ios::out);
 
-  int start = 0;
-  if (history.size() > (size_t)HIST_MAX_SIZE)
-    start = history.size() - HIST_MAX_SIZE;
+    int start = 0;
+    if (history.size() > (size_t)HIST_MAX_SIZE)
+      start = history.size() - HIST_MAX_SIZE;
 
-  for (size_t i = start; i < history.size(); i++)
-    if (history[i].length() != 0)
-      f << history[i] << endl;
+    for (size_t i = start; i < history.size(); i++)
+      if (history[i].length() != 0)
+        f << history[i] << endl;
 
-  f.flush();
+    f.flush();
+  }
 }
 
 void
